internal/apps/dop/providers/issue/core/provider.go

```diff
@@ -160,10 +160,19 @@ type IssuePermProjectID interface {
 	GetProjectID() uint64
 }
 
+type IssuePermProjectIDint64 interface {
```

Comment:
strange name ends with `int64`.
maybe `IssuePermProjectIDFetcher` is better.

Reply:
done